### ISSUE TITLE
svgo breaks image

### ISSUE THREAD

**Issue by ikokostya** [svg/svgo#151]
```
svgo -v
0.4.1
```

Original:
![slice 1](https://f.cloud.github.com/assets/328125/1621035/d8ee58a2-5684-11e3-84d6-9cbb5f46ff98.png)

```svg
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg width="25px" height="25px" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:sketch="http://www.bohemiancoding.com/sketch/ns">
    <title>Slice 1</title>
    <description>Created with Sketch (http://www.bohemiancoding.com/sketch)</description>
    <defs></defs>
    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" sketch:type="MSPage">
        <path d="M12.5,25 C19.4035597,25 25,19.4035597 25,12.5 C25,5.59644028 19.4035597,0 12.5,0 C5.59644028,0 0,5.59644028 0,12.5 C0,19.4035597 5.59644028,25 12.5,25 C12.5,25 5.59644028,25 12.5,25 Z M12.5,22.1111111 C17.7467054,22.1111111 22,17.8578165 22,12.6111111 C22,7.36440572 17.7467054,3.11111111 12.5,3.11111111 C7.25329461,3.11111111 3,7.36440572 3,12.6111111 C3,17.8578165 7.25329461,22.1111111 12.5,22.1111111 C12.5,22.1111111 7.25329461,22.1111111 12.5,22.1111111 Z M12.5,22.1111111" id="Oval-3-copy-2" fill="#BEBEBE" sketch:type="MSShapeGroup"></path>
        <ellipse id="Oval-1" fill="#BEBEBE" sketch:type="MSShapeGroup" cx="8.5" cy="9.55555556" rx="1.5" ry="1.55555556"></ellipse>
        <ellipse id="Oval-1-copy" fill="#BEBEBE" sketch:type="MSShapeGroup" cx="16.5" cy="9.55555556" rx="1.5" ry="1.55555556"></ellipse>
        <path d="M12.5,15 C10.5,15 7.95385361,15.7753488 7.95385361,15.7753488 C7.42705481,15.9198752 7,16.4768948 7,17.0376224 L7,17.1105257 C7,17.6631338 7.43507367,17.988077 7.96417882,17.8467396 C7.96417882,17.8467396 10.5,17.0740741 12.5,17.0740741 C14.5,17.0740741 17.0461464,17.8494229 17.0461464,17.8494229 C17.5729452,17.9939493 18,17.6712533 18,17.1105257 L18,17.0376224 C18,16.4850144 17.5649263,15.9140029 17.0358212,15.7726656 C17.0358212,15.7726656 14.5,15 12.5,15 Z M12.5,15" id="Rectangle-1" fill="#BEBEBE" sketch:type="MSShapeGroup"></path>
    </g>
</svg>
```

After svgo:
![new](https://f.cloud.github.com/assets/328125/1621037/e20cd15c-5684-11e3-88c2-ca0083654151.png)

**Comment by GreLI** [comment]
Actually it's ok now.

**Comment by pruett** [comment]
i'm experiencing similar issues with sketch exported .svg files, sorry if i misunderstand, but what exactly is `ok now`?

**Comment by GreLI** [comment]
Yes, such a bugs were fixed in v0.5.1. Are you using a recent version?

**Comment by pruett** [comment]
i am, i'll open another ticket and provide detailed information and reference this issue. thanks @GreLI
